pkg/lib/editor/controllers.go

```diff
@@ -210,6 +210,16 @@ func validateConfigBundle(opts *ServerOptions) func(http.ResponseWriter, *http.R
 			Certificates: configBundle.Certificates,
 		}
 
+		// Write database.pem needed for db validation, if any
+		if database_pem_bytes, ok := configBundle.Certificates["database.pem"]; ok {
```

Comment:
linter is complaining here with: "don't use underscores in Go names; var database_pem_bytes should be databasePemBytes"

Reply:
Fixed